tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3955,19 +3955,42 @@ declare namespace ts {
      * A function that transforms a node.
      */
     export type Transformer<T extends Node> = (node: T) => T;
+    export type VisitResult<T extends Node | undefined, U extends Node = NonNullable<T>> = T | readonly U[];
```

Comment:
It seems like you don't need the second type parameter, do you?

Reply:
Yeah. this can probably just be `Node[]` and that's fine. The lift function does not care about the node type.